.github/workflows/libxrpl.yml

```diff
@@ -0,0 +1,85 @@
+name: Upload libXRPL to Conan
+env:
+  CONAN_URL: http://18.143.149.228:8081/artifactory/api/conan/conan-non-prod
+  CONAN_LOGIN_USERNAME_RIPPLE: ${{ secrets.CONAN_USERNAME }}
+  CONAN_PASSWORD_RIPPLE: ${{ secrets.CONAN_TOKEN }}
+on:
+  pull_request:
+    paths:
+      - 'src/ripple/protocol/impl/BuildInfo.cpp'
+      - '.github/workflows/libxrpl.yml'
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  publish:
+    name: Publish libXRPL
+    outputs:
+      outcome: ${{ steps.upload.outputs.outcome }}
+      version: ${{ steps.version.outputs.version }}
+    runs-on: [self-hosted, heavy]
+    container: rippleci/rippled-build-ubuntu:aaf5e3e
+    steps:
+      - name: Wait for essential checks to succeed
+        uses: lewagon/wait-on-check-action@v1.3.4
+        with:
+          ref: ${{ github.event.pull_request.head.sha || github.sha }}
+          running-workflow-name: wait-for-check-regexp
+          check-regexp: '(dependencies|test).*linux.*' # Ignore windows and mac tests but make sure linux passes
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          wait-interval: 10
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Unlock Conan
+        shell: bash
+        run: conan remove --locks
```

Comment:
The environment should always be reset after each job, so `conan` shouldn't be locked. Has this been an issue?

Reply:
I don't know tbh., this is copy-pasted from the `dependencies/action.yml`. I assume it was needed but perhaps it is only needed when used through the action.

Reply:
I verified - it works fine without. Removed now 👍

Reply:
It was needed on macOS. Maybe still is. Like I said, after this goes in, the workflows are ripe for refactoring.